src/snowflake/cli/plugins/spcs/image_registry/manager.py

```diff
@@ -45,3 +53,13 @@ def login_to_registry(self, repo_url):
         if resp.status_code != 200:
             raise ClickException(f"Failed to login to the repository {resp.text}")
         return json.loads(resp.text)["token"]
+
+    def get_registry_url(self):
+        repositories_query = "show image repositories in account"
+        result_set = self._execute_query(repositories_query, cursor_class=DictCursor)
+        results = result_set.fetchall()
+        if len(results) == 0:
+            raise NoRepositoriesViewableError()
+        sample_repository_url = results[0]["repository_url"]
+
+        return "/".join(sample_repository_url.split("/")[:-3])
```

Comment:
maybe use urllib.parse or add some validation that this is in the expected format (ends with snowflakecomputing.com)